### ISSUE TITLE
Create a Logo

### ISSUE THREAD

**Issue by TurkeyMan** [FeedBackDevs/feedback#1]
Needs a logo + branding.
I chose the name 'FeedBack' because it seems applicable in the context of a rock-oriented music game, and also I saw some cool branding opportunities in the 'dB' appearing as the middle 2 letters of the name...

Any artists floating around?

**Comment by thegman666** [comment]
Any symbols or anything you want in the logo? Any font preferences?

**Comment by TurkeyMan** [comment]
Wow. 3 years later! :)

Yeah, I have no strong ideas. I wouldn't want to bias creativity, other than the fact I think the central letters dB are a cute feature; decibel being the measure for volume/loudness ;)
I've used dB as the abbreviation for FeedBack since the start.

An obvious requirement is that modern logo's need to fit nicely into a square app icon for phones/appstores, but also scale to a nice horizontal-spanning fullscreen in-game logo.

And logically, it should be styled such that the look&feel of the default skin/theme can be derived. It's probably more practical to imagine the default skin/theme, and derive the logo from that actually...

I wonder if google images might present some ideas:
https://www.google.com.au/search?q=decibel&biw=1920&bih=1070&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjbjb337OjRAhVBX5QKHdJ1Ak8Q_AUIBigB#tbm=isch&q=decibel+symbol
https://www.google.com.au/search?q=decibel&biw=1920&bih=1070&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjbjb337OjRAhVBX5QKHdJ1Ak8Q_AUIBigB#tbm=isch&q=decibel+meter

**Comment by thegman666** [comment]
Cool, thanks for getting back to me so quickly! i do like the design of the [decible meter](https://thumbs.dreamstime.com/z/sound-level-meter-26621414.jpg), so maybe we could work that in?

**Comment by TurkeyMan** [comment]
Yeah, that's where my mind started, and influenced the name... like, over 10 years ago ;)

**Comment by thegman666** [comment]
Haha, cool, i have to reinstall GIMP, so after that I'll work on this.

**Comment by TurkeyMan** [comment]
I also love nerdy and in-jokes... like, if there's a meter, it has to 'go to 11', and all that sort of thing :)

**Comment by thegman666** [comment]
with tape over 12, as a reference to the intro screens?

**Comment by TurkeyMan** [comment]
It's a [spinal tap reference](https://www.youtube.com/watch?v=4xgx4k83zzc) ;)

**Comment by thegman666** [comment]
Yeah, the intro cinematic to gh3 has an amp like that, with tape over the 11 saying 12 which is what I was saying. Also, there's an [xkcd](https://xkcd.com/670/) about it.

**Comment by TurkeyMan** [comment]
Hah! Man, I can't believe I missed that!

**Comment by thegman666** [comment]
And the intro sequence is [here](https://youtu.be/zFrflmw3Iag?t=22s).

**Comment by thegman666** [comment]
https://www.mediafire.com/?zr2paa4asy4rnnl is my first version.
